components/31-molecules/file-upload/file-upload--multiple.twig

```diff
@@ -1,6 +1,4 @@
-<fieldset class="file-upload">
-  <legend>{{ label }}</legend>
-
+<section class="file-upload multiple">
```

Comment:
We proberen zoveel mogelijk de html markup ongewijzigd te laten tussen stjilgids v5 en v6 om de overgang zo soepel mogelijk te maken in Drupal en andere apps, alleen is dit niet altijd mogelijk. Alle wijzigingen lijsten we op in de CHANGELOG.md. Je kan je baseren op de bestaande changelog notities.
Zodra we implementeren in Drupal kunnen we deze changelog gebruiken om de nodige markup wijzigingen door te voeren.
Het komt erop neer dat een ontwikkelaar die de nieuwe stijlgids v6 implementeert een duidelijke lijst ziet van aan te passen HTML markup in zijn applicatie waar dan de nieuwe css op toegepast zal worden.

Reply:
Ik heb een extra note toegevoegd met uitleg van de verandering en waarom dit nodig was.